browser/src/dom/NotebookbarAccessibility.js

```diff
@@ -195,13 +198,37 @@ var NotebookbarAccessibility = function() {
 					this.state = 1;
 				}
 				else if (this.filteredItem.focusBack === true) {
-					app.map.focus();
+					this.focusToMap();
 				}
 			}
 			this.filteredItem = null;
 		}
 		else
-			app.map.focus();
+			this.focusToMap();
+	};
+
+	this.addTabFocus = function() {
+		var element = this.getCurrentSelectedTab();
+		if (element) {
+			element.classList.add('add-focus-to-tab');
```

Comment:
can't we just make the browser using the `:focus-visible` pseudo class to the element by, when the arrow keys are pressed we
1. `element.focus();`
2. Then switch tab

?

Plus I'm also getting sometimes that border even when the focus is already in the document and the active tab is a different one

Reply:
yes we can make direct focus to tab elment by foucs but if we do that then  accessibility info keys will be focus out and in simple case we lost all info tags like `N` for insert because focus shifted to tab.

Reply:
actully if you see this addTabFocus is only used when we have focus to accessibilty keys. 

we need 2 things at a time focus should be on accessibility keys and
that active tab should highlighted 

so to add explicit focus thing on element at same time i have introduced this class